cg/constants/demultiplexing.py

```diff
@@ -39,50 +56,90 @@ class RunParametersXMLNodes(StrEnum):
     UNKNOWN_REAGENT_KIT_VERSION: str = "unknown"
 
 
-class SampleSheetHeaderColumnNames(StrEnum):
-    DATA: str = "[Data]"
-    FLOW_CELL_ID: str = "FCID"
-    LANE: str = "Lane"
-    SAMPLE_INTERNAL_ID: str = "Sample_ID"
-    SAMPLE_NAME: str = "SampleName"
-    SAMPLE_PROJECT: str = "Sample_Project"
-    CONTROL: str = "Control"
-
-
-SAMPLE_SHEET_HEADERS = {
-    "bcl2fastq": [
-        SampleSheetHeaderColumnNames.FLOW_CELL_ID,
-        "Lane",
-        "SampleID",
-        "SampleRef",
-        "index",
-        "index2",
-        "SampleName",
-        "Control",
-        "Recipe",
-        "Operator",
-        "Project",
-    ],
-    "dragen": [
-        SampleSheetHeaderColumnNames.FLOW_CELL_ID,
-        "Lane",
-        "Sample_ID",
-        "SampleRef",
-        "index",
-        "index2",
-        "SampleName",
-        "Control",
-        "Recipe",
-        "Operator",
-        "Sample_Project",
-    ],
-}
+class SampleSheetNovaSeq6000Sections:
```

Comment:
Maybe `SampleSheetNovaSeq6000Constants` is a better name?

Reply:
I decided to name it Sections as this class holds group constants by sample sheet sections in subclasses. I think the word constants is unnecessary as they are located in a script where all demux constants are stored